src/screens/UserPortal/Settings/Settings.tsx

```
@@ -56,179 +82,126 @@
       [fieldName]: value,
     }));
   };
 
-  /**
-   * Triggers the file input click event to open the file picker dialog.
-   */
-  const handleImageUpload = (): void => {
-    setisUpdated(true);
-    if (fileInputRef.current) {
-      (fileInputRef.current as HTMLInputElement).click();
+  const fileUploadHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)